packages/governance/src/contractGovernance/typedParamManager.js

```diff
@@ -66,6 +66,8 @@ const isAsync = {
  * | ST<'nat'>
  * | ST<'ratio'>
  * | ST<'string'>
+ * | ST<'timestamp'>
+ * | ST<'relativeTime'>
```

Comment:
Intentional to omit the other new param type `'timerService'`?

`ParamValueForType` needs these cases too.